packages/components/stamp/src/stamp.tsx

```diff
@@ -40,63 +41,84 @@ const getPaddingStyle = (props: Props) => {
   `;
 };
 
-const getToneStyles = (props: Props) => {
+const getToneStyles = (props: Props & { isNewTheme: boolean }) => {
   switch (props.tone) {
     case 'critical': {
       return css`
         background-color: ${designTokens.colorError95};
-        border: 1px solid ${designTokens.colorError};
+        border: 1px solid
+          ${props.isNewTheme
+            ? designTokens.borderForStampWhenError
+            : designTokens.colorError};
```

Comment:
If we use design tokens, we don't need to use a variable. You should override the design token value for the new theme.